### ISSUE TITLE
Handle XML parsing errors

### ISSUE THREAD

**Issue by ChristianKuehnel** [google/llvm-premerge-checks#124]
https://jenkins.llvm-merge-guard.org/job/amd64_debian_testing_clang8/2323/console

1. Figure out why the xml parser failed.
2. Make sure we still give some build feedback in that case.

**Comment by metaflow** [comment]
Yes, I saw that error and added 'du test-results.xml' right after copy is suppose to be finished.
It reports that file size is only 5KB:
```
copying test_results.xml to /mnt/nfs/results/amd64_debian_testing_clang8-2323
5892	test-results.xml
```
while https://results.llvm-merge-guard.org/amd64_debian_testing_clang8-2323/ shows that file is 6MB.
My guess is that despite `cp` has returned the file is still being transferred to NFS and python observes only half of it.
Plausible?

**Comment by ChristianKuehnel** [comment]
fixed with 22ce8e96326f97de1963b79949a98da95ade78c1
